src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -190,6 +208,9 @@ class SuggestedOwners extends React.Component {
             <GuideAnchor target="owners">
               <h6>
                 <span>{t('Ownership Rules')}</span>
+                <Hovercard body={this.getHovercardBody()} containerClassName="pill-icon">
```

Comment:
Not sure if this should have the class `pill-icon`

Reply:
should it be something else?  or remove it?

Reply:
removed it.